### DISCUSSION TITLE
JoTL scenario #3, item #27

### DISCUSSION THREAD

**Discussion by ArmadilloBrillo** [Lurkars/gloomhavensecretariat#280] [Q&A] [answered]
Excellent job in GHS!

I've started using it, removing a lot of housekeeping and tokens from the game and it's great.
I have 1 question, not sure if it's something I am missing or an actual issue.

Looting the chest in JoTL scenario #3 gave me item #27 (loot index #10) but I can't seem to add it to the character that looted it.
Is this a bug or am I overlooking some game mechanic?

**Comment by Lurkars** [comment, marked as answer]
The item should be just present on the corresponding character sheet. (Open character sheet by clicking on the big character icon on the right)

**Reply by Lurkars** [reply]
![IMG_20230507_152948.jpg](https://user-images.githubusercontent.com/2760570/236680488-9364d078-c022-44ad-b228-a84ac381703e.jpg)

![IMG_20230507_153007.jpg](https://user-images.githubusercontent.com/2760570/236680479-5dd7bcf1-231f-4df3-9960-bd6eb2853961.jpg)

**Reply by ArmadilloBrillo** [reply]
Thanks for the quick reply.

I am running GHS locally in a docker container and it seems that the loot assignment (clicking on the scenario name in the bottom) isn't working the same for me as it does in the online version.

The online version let's me assign who looted a box and, as per your screenshots, I can indeed see the item appear on the relevant character sheet. My locally running version does not allow me to assign in the same way.  The treasures icon is greyed out when clicking the scenario name in the bottom so i can't assign it.

Only difference I can see is that I am running 0.63 whereas the online version is 0.62. Not sure it's my setup. 

I'll dig around a bit the coming days to see if it's some weirdness in my container :-)
I could always switch to the online version though.

Thanks for the help.
I am quite new to the tool so I am still figuring out how some of these things work.

**Reply by Lurkars** [reply]
Not possible that the online version is older than your docker, so possible, that you just have a cached online version, anyways both v.062.0 and v.63.0 should behave completely the same regarding treasures and stuff. A greyed out boxed just mean that all revealed treasures are already looted. I really cannot think of a scenario where your docker version can behave differently. Maybe check if all settings and setup is really same. There is for example setting `Apply treasure reward` which should be checked by default, but you could accidentally unchecked it and just disabled the feature you're asking for. So please compare your setup with a working online version. Pretty sure, it will behave correctly when all same!

**Comment by ArmadilloBrillo** [comment]
Check. That was it. Sorry for any confusion 👍
